brc_pytorch/layers/multilayer_rnnbase.py

```diff
@@ -1,199 +1,330 @@
-from typing import Tuple
-
+from typing import Tuple, List
+import copy
 import torch
 import torch.nn as nn
 
 
 class MultiLayerBase(nn.Module):
-    """Multi layer forward recurrent network."""
+    """Executes a multilayer recurrent network given RNN cells for each layer."""
 
     def __init__(
         self,
-        mode,
-        cells,
-        hidden_sizes: int,
-        device: str = 'cpu',
+        mode: str,
+        cells: List,
```

Comment:
here u can be more explicit `List[torch.nn.Module]` and then mention in docstring e.g. `torch.nn.GRUCell` so that logic is clear